Maths/PowLogarithmic.js

```diff
@@ -0,0 +1,13 @@
+import { isEven } from './IsEven'
+
+// Returns the value of x to the power of n (x^n)
```

Comment:
Add a description for your algorithm, like how it works and how it is different.

Reply:
Done